workflows/bacterial_genomics/bacterial_genome_assembly/bacterial_genome_assembly.ga

```diff
@@ -0,0 +1,705 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Short paired-end read analysis to provide assembly quality metrics",
```

Comment:
Maybe sth like "Assembly of bacterial paired-end short read data with generation of quality metrics and reports"?